components/main.js

```diff
@@ -24,6 +24,7 @@ require(['./config'], function(config) {
     require(['angular', 'my-app'], function(angular) {
         angular.bootstrap(document, ['my-app']);
         // apply cssvars ponyfill for IE11
+        // eslint-disable-next-line no-undef
```

Comment:
It could also be added to the list of globals :man_shrugging:
https://eslint.org/docs/user-guide/configuring#specifying-globals